idb/postgres/postgres.go

```diff
@@ -39,6 +40,9 @@ const stateMetastateKey = "state"
 const migrationMetastateKey = "migration"
 const specialAccountsMetastateKey = "accounts"
 
+// Be a real ACID database
+var serializable = sql.TxOptions{Isolation: sql.LevelSerializable}
```

Comment:
I'm confused at how this helps. I thought we need to enable dirty reads with `sql.LevelReadUncommitted` in order to see if multiple transactions are working on the same data concurrently.

Reply:
Looks like it will fail when each transaction tries to write the same thing with different values. Nice.